docs/proposals/YYYYMMDD-template.md

```diff
@@ -6,7 +6,7 @@ reviewers:
   - "@janedoe"
 creation-date: yyyy-mm-dd
 last-updated: yyyy-mm-dd
-status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+status: provisional|experimental|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
Do we need a different category for `experimental` vs (not sure what to call non-experimental here)?

It seems like we'd want to be able to track `provisional` vs `implementable` vs `implemented`, etc independent of if it is experimental or not. That said, how would promotion out of experimental work in that case? A new CAEP to replace the experimental CAEP or swapping the experimental bit and going back to provisional? Either seems a bit odd for a workflow.

Alternatively, maybe we could duplicate `implementable`/`implemented` and have a `experimental-implementable` and `experimental-implemented` status that could then a workflow that goes through experimental could look like this:

`provisional` -> `experimental-implementable` -> `experimental-implemented` -> `implementable` -> `implemented` 

or:

`provisional` -> `experimental-implementable` -> `experimental-implemented` -> `rejected`

Reply:
As an aside, we likely need to track following up and updating the existing CAEPs.

Reply:
I'd rather just keep it simple for now and keep only `experimental` for now, promotion process for the experiments should be outlined separately

Reply:
If the lifecycle suggestions I added above sound good, we might want to consider having 'early experimental' and 'late experimental' here and updating the lifecycle to indicate that the CAEP should be in the appropriate state prior to moving to that stage of the lifecycle.